docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -44,7 +44,74 @@ Add an introductory paragraph here (include visual of 'What this is vs. what thi
 
 1.1 Guiding Principles
 ----------------------
-In this section, we will outline the guiding principles of BLISS.
+
+Our guiding principles for this document can be summarized in three points:  
+
+**1. This work is important**
+
+Inclusive language use is important, and we want to begin by acknowledging the 
+potential harm that using gendered language can create. ‘Sex’ and ‘gender’ are 
+not synonymous, unidimensional traits. [Bauer_2022]_ acknowledges this and explains 
+how common sex and gender proxies such as chromosomes or sex organs are imperfect 
+and not fully inclusive of lived experiences.  
+
+Using imperfect proxies can create real harm by alienating individuals, and by 
+creating invalid research. People who are not cisgendered (a diverse group including 
+intersex, nonbinary, transgender and other gender nonconforming individuals) experience 
+gendered language frequently which can be alienating and cause harm. These individuals 
```

Comment:
I think this last sentence (starting with 'People who are not cis...') could actually be removed, it feels a little redundant to me.

Reply:
Fair! I combined with with the sentence below and shortened them to just include the examples rather than statement and then separately examples.